lib/typeUtil.js

```diff
@@ -284,6 +285,56 @@ function clone(oldItem) {
   }
 }
 
+/**
+ * @see http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ExpressionPlaceholders.html#ExpressionAttributeNames
+ * @return {?string} The alias if we need one, null otherwise.
+ */
+function getAttributeAlias(key) {
```

Comment:
Wonder if it would make the rest of the code easier if we have an alias map for all the attributes. If an attribute is not a reserve word, its alias is just itself.